### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by mynumu** [ChatGPTNextWeb/NextChat#4029] [General]
Hi - love this - can it generate images? Tried all the openai vision models and it says it can only do text thanks

**Comment by H0llyW00dzZ** [comment]
- ChatGPTNextWeb#3378

**Reply by mynumu** [reply]
aha i see you have a forked repo - is there an official release??

**Reply by H0llyW00dzZ** [reply]
nah I am lazy to made it official release since it only fork and web services

**Reply by H0llyW00dzZ** [reply]
@mynumu I've been improved a image generation for DALL-E, should be better now instead of before it much complex

**Comment by fred-bf** [comment]
We are currently supporting multimodal models, including Dalle. In the future, it will be possible to use GPT Vision directly or Dalle via plugins. This is expected to be released in the next major version.

**Reply by mynumu** [reply]
hi ok thanks - if you support multimodal now - whats the best way to get this up and running as I dont see that option. I see that hollywoodz seems to have another repo??Thanks

**Reply by fred-bf** [reply]
NextChat doesnt support multimodal model right now. If you're in a hurry, you can consider using a different fork version first
